clean-deployments.groovy

```diff
@@ -0,0 +1,16 @@
+node('spin-k8s-test') {
```

Comment:
I was hoping to add tests for the other quickstart templates (basically one test for each to at least make sure the deployment succeeds). So I will end up with quite a few 'deploy' jobs and I wanted one separate 'clean' job that cleans them all